### PR TITLE
Fix productpage, mysqldb and bump images version

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
* bump the images version to 1.14.0
* fix the egress rule to google apis, to rewrite port from 80 to 443 while performing TLS origination
* undelete lines unintentionally deleted by a previous merge
* use TCP as a protocol for the mysql port, otherwise it does not work, mentioned in https://github.com/istio/istio/issues/10062#issuecomment-489516364
* use the old default mysql authentication plugin mysql_native_password.
The new plugin, caching_sha2_password is not yet supported by node.js.
The situation is described in this Stack Overflow answer - https://stackoverflow.com/a/50377944.

### CODE REVIEW

File: samples/bookinfo/networking/egress-rule-google-apis.yaml [hunk 1]
```diff
@@ -4,18 +4,10 @@ metadata:
...
-metadata:
-  name: googleapis
-spec:
-  host: "*.googleapis.com"
-  trafficPolicy:
-    tls:
-      mode: SIMPLE # initiates HTTPS when talking to www.google.com
\ No newline at end of file
+  location: MESH_EXTERNAL
+  resolution: DNS
```
Comment (howardjohn) [inline samples/bookinfo/networking/egress-rule-google-apis.yaml]: nit: Don't need DNS here I think

We are planning on cleaning up unneeded usages of DNS resolution in the samples and docs

Comment (vadimeisenbergibm) [reply samples/bookinfo/networking/egress-rule-google-apis.yaml]: OK, I will remove it.

Comment (vadimeisenbergibm) [reply samples/bookinfo/networking/egress-rule-google-apis.yaml]: @howardjohn I just checked, for TLS origination DNS is required for some reason. Note that www.googleapis.com is used as a destination in the virtual service. (The virtual service is required to rewrite port 80 to 443).

Comment (howardjohn) [reply samples/bookinfo/networking/egress-rule-google-apis.yaml]: Great, good to know. I'll have to look more into that. Let me know when this isn't a WIP anymore

Comment (vadimeisenbergibm) [reply samples/bookinfo/networking/egress-rule-google-apis.yaml]: @howardjohn The PR is ready, please review it.

**Comment by linsun** [comment]
@vadimeisenbergibm do we need this in 1.2 branch?

**Comment by zeph** [comment]
@linsun would be nice, I just put my feet into this nice `mysql vs tcp` trap

**Pull request merged**